src/agent/Cargo.toml

```diff
@@ -7,6 +7,7 @@ members = [
     "onefuzz-agent",
     "onefuzz-supervisor",
     "storage-queue",
+    "utils",
```

Comment:
Can this go in a `retry` module of the `onefuzz` crate?

Reply:
I will have to reference onfuzz in the storage-queue and proxy-managed crates

Reply:
can it be named something other than utils then?  we have a large number of `utils` spread across multiple crates as is.